tests/test_ind_indygo_service.py

```diff
@@ -2,7 +2,7 @@
 from os.path import dirname, join
 
 import pytest
-from city_scrapers_core.constants import BOARD
+from city_scrapers_core.constants import BOARD  # noqa
```

Comment:
See this [comment](https://github.com/City-Bureau/city-scrapers-indianapolis/pull/14/files#r1463728936), re: suppressing linting issues vs resolving them. Although I know it's tempting to ignore the warning, flake8 is our friend and we should try to listen to him (even though he can be awfully whiny about certain things) 😄